components/ModerateSubmissionsEditForm.vue

```diff
@@ -0,0 +1,97 @@
+                />
+                </div>
+            </div>
+
+            <div v-else-if="chosenFormName == 'Address'">
+                <span class="mb-3.5 text-center text-zinc-900 text-2xl font-bold font-['Noto Sans JP'] leading-normal">
+                Addresses
+                </span>
+                <div v-for="(field, index) in addressesFields" :key="index" class="flex flex-col">
+                <label class="mb-2 text-zinc-900 text-sm font-normal font-['Noto Sans JP']">
+                    {{ field }}
+                </label>
+                <input 
+                    type="text" 
+                    class="mb-5 px-3 py-3.5 w-[350px] h-[50px] bg-white rounded-lg border border-zinc-400 text-neutral-600 text-sm font-normal font-['Noto Sans JP']"
+                />
+                </div>
+            </div>
+            
+            <div v-else-if="chosenFormName == 'Google maps information'">
+                <span class="mb-3.5 text-center text-zinc-900 text-2xl font-bold font-['Noto Sans JP'] leading-normal">
+                Google maps information
+                </span>
+                <div v-for="(field, index) in googleMapsFields" :key="index" class="flex flex-col">
+                <label class="mb-2 text-zinc-900 text-sm font-normal font-['Noto Sans JP']">
+                    {{ field }}
+                </label>
+                <select 
+                    v-if="field == 'Healthcare professional IDs'"
+                    class="mb-5 px-3 py-3.5 w-[350px] h-[50px] bg-white rounded-lg border border-zinc-400 text-neutral-600 text-sm font-normal font-['Noto Sans JP']"
+                >
+                    <option value="Example 1">Example 1</option>
+                    <option value="Example 2">Example 2</option>
+                    <option value="Example 3">Example 3</option>
+                </select>
+                <input 
+                    v-else
+                    type="text" 
+                    class="mb-5 px-3 py-3.5 w-[350px] h-[50px] bg-white rounded-lg border border-zinc-400 text-neutral-600 text-sm font-normal font-['Noto Sans JP']"
+                />
+                </div>
+            </div>
+
+            <div v-else-if="chosenFormName == 'Healthcare professional IDs'">
+                <span class="mb-3.5 text-center text-zinc-900 text-2xl font-bold font-['Noto Sans JP'] leading-normal">
+                    Healthcare professional IDs
+                </span>
+            </div>
+
+            <div v-else-if="chosenFormName == 'Change Log'">
+                <span class="mb-3.5 text-center text-zinc-900 text-2xl font-bold font-['Noto Sans JP'] leading-normal">
+                    Change Log
+                </span>
+            </div>
+
+        </form>
+    </div>
+</template>
+
+<script lang="ts" setup>
+import {ref} from 'vue'
+
+let chosenFormName = ref<string>('Contact information')
```

Comment:
let's i18n'ify these! Can you move these names to the i18n json files? This will allow headers in different languages